code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -179,10 +179,11 @@
 		on = FALSE
 		update_icon()
 		playsound(src, 'sound/machines/cryo_warning.ogg', volume) // Bug the doctors.
-		radio.talk_into(src, "Patient fully restored", radio_channel, get_spans(), get_default_language())
+		var/msg = "Patient fully restored."
 		if(autoeject) // Eject if configured.
-			radio.talk_into(src, "Auto ejecting patient now", radio_channel, get_spans(), get_default_language())
+			msg = "[msg] Auto ejecting patient now."

```

Comment:
```suggestion
			msg += " Auto ejecting patient now."
```

Reply:
Does it matter

Reply:
+ operator is faster, as [] operator formats the inner content

Reply:
also it follows convention better, we do it this way in the rest of our code